### ISSUE TITLE
Which branch should I track?

### ISSUE THREAD

**Issue by sodabrew** [wkhtmltopdf/qt#3]
I see current commits on both wk_4.8 and wk_4.8.5. Which one should I be testing/developing against?

**Comment by ashkulz** [comment]
`wk_4.8.5` is based on QT 4.8.5 and up until the 0.12.0 release I was rebasing it regularly as and when required (sometimes, fixes were found in previous commits -- instead of adding a new commit I just squashed the changes into the existing commit). After the release, I stopped doing that as the branch is now "stable" and tagged -- various distributions may use the tag to pull the patched QT version (FreeBSD is definitely doing that for sure).

`wk_4.8` is tracking changes in the upstream 4.8 branch, which is shortly to become QT 4.8.6. I rebase it regularly and squash commits as required, so that when 4.8.6 is released we have almost no effort to move to that version. Additionally, the Mac OS X builds currently work only with `wk_4.8` due to fixes in upstream QT -- @npinchot will be trying to find out the exact fixes to backport if he finds the time. When QT releases 4.8.6 I will be making a `wk_4.8.6` branch for which the same rules are applied, and leave `wk_4.8` tracking the upstream 4.8 repository.

So it is up to you to decide which branch to track .. the stable (`wk_4.8.5`) or the upcoming release (`wk_4.8`). It should have been [released in January](https://www.mail-archive.com/development@qt-project.org/msg12805.html), but I guess it got delayed -- hopefully it will [happen in February](http://lists.qt-project.org/pipermail/development/2014-January/015294.html).
